gssapi_kerberos.go

```diff
@@ -33,6 +33,7 @@ type GSSAPIConfig struct {
 	Username           string
 	Password           string
 	Realm              string
+	DisablePAFXFAST    bool
```

Comment:
I'm seeing some test for this API here `sarama/kerberos_client_test.go`, might worth to add a test that includes `DisablePAFXFAST`